components/brave_rewards/resources/page/reducers/wallet_reducer.ts

```diff
@@ -158,6 +167,13 @@ const walletReducer: Reducer<Rewards.State | undefined> = (state: Rewards.State,
       }
 
       state.externalWallet = action.payload.wallet
+      if (action.payload.openVerifyUrl && state.externalWallet != undefined) {
+        let verifyUrl = state.externalWallet.verifyUrl
+
+        if (verifyUrl != "") {
```

Comment:
Nit: Single quotes and should use `!==`. You might need to `npm run tslint` to catch all of these.

Reply:
resolved